plugins/Monolog/Formatter/LineMessageFormatter.php

```diff
@@ -53,6 +53,8 @@ public function format(array $record)
         $total = '';
 
         foreach ($messages as $message) {
+            // escape control characters
+            $message = addcslashes($message, "\x00..\x1F\x7F");
```

Comment:
Do we need to treat `\x0A` (line break) at bit more special based on `$allowInlineLineBreaks`?

Like using `"\x00..\x09\x0B..\x1F\x7F"` when `$this->allowInlineLineBreaks === true`?

Reply:
Guess it makes sense to exclude them there at all, as line endings are already handled above